### PR TITLE
Fixes #20150 by updating global-json.md to have cleaner info on match…

### LINKED ISSUE #20150: Specify what the default rollForward value is if not specified
Would be great to know what the default rollForward value is if not specified. 

Would've thought it would be latestFeature, but it would seem not, so probably just latestPatch, which makes sense for security reasons, but isn't 100% obvious when the dotnet versioning is `x.y.znn` and z = feature bands. Whereas semantic versioning usually would have that last `znn` part be the whole patch version.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a94c335b-0b65-96cc-1c25-65777fbe4bb5
* Version Independent ID: 6d80613f-42dc-35d7-aeff-3187a36dbbca
* Content: [global.json overview - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x)
* Content Source: [docs/core/tools/global-json.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/global-json.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by eriawan**
## Summary

Changes are based on suggestions on https://github.com/dotnet/docs/issues/20150#issuecomment-676005236

Fixes #20150

cc  @tdykstra

### CODE REVIEW

File: docs/core/tools/global-json.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ If you don't set this value explicitly, the default value depends on whether you
 - Available since: .NET Core 3.0 SDK.
 
 The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a higher version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`.
+The default value of `rollForward` is determined by the [matching rules](https://docs.microsoft.com/dotnet/core/tools/global-json?tabs=netcore3x#matching-rules).
```
Comment (Youssef1313) [inline docs/core/tools/global-json.md:61]:
```suggestion
The default value of `rollForward` is determined by the [matching rules](#matching-rules).
```

Comment (eriawan) [reply docs/core/tools/global-json.md:61]: Thanks!

File: docs/core/tools/global-json.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ If you don't set this value explicitly, the default value depends on whether you
 - Available since: .NET Core 3.0 SDK.
 
 The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a higher version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`.
-The default value of `rollForward` is determined by the [matching rules](https://docs.microsoft.com/dotnet/core/tools/global-json?tabs=netcore3x#matching-rules).
+The default value of `rollForward` is determined by the [matching rules](#matching-rules).
```
Comment (tdykstra) [inline docs/core/tools/global-json.md:61]:
```suggestion
The default value roll forward behavior is determined by the [matching rules](#matching-rules).
```

File: docs/core/tools/global-json.md [hunk 3]
```diff
@@ -157,7 +158,7 @@ dotnet new globaljson --sdk-version 3.0.100
 ## Matching rules
 
 > [!NOTE]
-> The matching rules are governed by the `dotnet.exe` entry point, which is common across all installed .NET Core installed runtimes. The matching rules for the latest installed version of the .NET Core Runtime are used when you have multiple runtimes installed side-by-side.
+> The matching rules are governed by the `dotnet.exe` entry point, which is common across all installed .NET Core installed runtimes. The matching rules for the latest installed version of the .NET Core Runtime are used when you have multiple runtimes installed side-by-side or if or you're using a global.json.
```
Comment (tdykstra) [inline docs/core/tools/global-json.md:161]:
```suggestion
> The matching rules are governed by the `dotnet.exe` entry point, which is common across all installed .NET Core installed runtimes. The matching rules for the latest installed version of the .NET Core Runtime are used when you have multiple runtimes installed side-by-side or if or you're using a *global.json* file.
```

**Review by tdykstra — COMMENTED** [review]
Thanks, looks good -- I made a couple of minor suggestions.

File: docs/core/tools/global-json.md (hunk 2 above)
Comment (eriawan) [reply docs/core/tools/global-json.md:61]: Thanks!

**Comment by eriawan** [comment]
Thanks @tdykstra 👍 

PR updated.

File: docs/core/tools/global-json.md [hunk 4]
```diff
@@ -58,6 +58,7 @@ If you don't set this value explicitly, the default value depends on whether you
 - Available since: .NET Core 3.0 SDK.
 
 The roll-forward policy to use when selecting an SDK version, either as a fallback when a specific SDK version is missing or as a directive to use a higher version. A [version](#version) must be specified with a `rollForward` value, unless you're setting it to `latestMajor`.
+The default value roll forward behavior is determined by the [matching rules](#matching-rules).
```
Comment (tdykstra) [inline docs/core/tools/global-json.md:61]: Looks like I accidentally left an extra word in my earlier suggestion.

```suggestion
The default roll forward behavior is determined by the [matching rules](#matching-rules).
```

**Comment by eriawan** [comment]
@tdykstra 
thanks! PR updated.

**Comment by eriawan** [comment]
@tdykstra 
Just to remind or to ask, is there anything I have to do?

**Comment by tdykstra** [comment]
No, I'll merge now.

**Review by tdykstra — APPROVED** [review]

**Pull request merged**